src/Screens/App/Screens/BooksDashboard/Screens/SearchBar/Screens/SearchComponent/style.css

```diff
@@ -0,0 +1,49 @@
+.search-component-container {
+  display: flex;
+  width: 240px;
+  height: 40px;
+  margin-right: 35px;
+  background: #f4f4f4;
+}
+
+.search-input {
+  display: flex;
+  flex-grow: 1;
+  appearance: none;
+  outline: 0;
+  border: 0 !important;
+  font-family: 'Roboto';
+  font-style: italic;
+  font-size: 14px;
+  letter-spacing: 0px;
+  background: #f4f4f4;
+  color: #9b9b9b;
+  padding-left: 12px;
+
+}
+
+::-webkit-input-placeholder { /* Chrome */
+  color: #9b9b9b;
+}
```

Comment:
I'm almost sure that this isn't necessary.
@sbalay thoughts ?

Reply:
The only way that I found to change a placeholder color is this:

https://stackoverflow.com/questions/2610497/change-an-html5-inputs-placeholder-color-with-css

I will change `::-webkit-input-placeholder` to `.search-input::-webkit-input-placeholder` to only modify the color of `.search-input`